### ISSUE TITLE
Doc needed regarding database migrations

### ISSUE THREAD

**Issue by Pierre-Michard** [Cryptonomic/Conseil#614]
When Updating Conseil binaries, it seems that the `lorre` process does not run database migrations automatically. 
Currently, the only option we have is to drop the current database and initialize a new one with the content of the  `doc/conseil.sql` file.
  - if this is the correct procedure, it would be nice to have this file in the docker image, so that we know which sql content to inject for a specific conseil docker image.
  - otherwise, it would be great to document how tu run database migrations, and eventually to have lorre process to run migrations before starting in the docker  image.

**Comment by vishakh** [comment]
Thanks @Pierre-Michard. You are right..we do not have a real upgrade process. Give us a couple of weeks and we will use this ticket to put one in.
